manala.phpredisadmin/templates/configs/default.j2

```diff
@@ -17,6 +17,8 @@ $config = array(
       {{ macros.config_row(config_server_config, 'host', '127.0.0.1') }},
       {{ macros.config_row(config_server_config, 'port', 6379) }},
       {{ macros.config_row(config_server_config, 'filter', '*') }},
+      {{ macros.config_row(config_server_config, 'scheme', 'tcp') }}, // Optional. Connection scheme. 'tcp' - for TCP connection, 'unix' - for connection by unix domain socket
+      {{ macros.config_row(config_server_config, 'path', '') }} // Optional. Path to unix domain socket. Uses only if 'scheme' => 'unix'. Example: '/var/run/redis/redis.sock'
```

Comment:
nitpicking: comments are aligned with each other below, we should do the same here